From f00bf8bc9e276f7372ea53fb09d6c51c07fc4b2c Mon Sep 17 00:00:00 2001
From: Benjamin Drung <benjamin.drung@profitbricks.com>
Date: Mon, 7 May 2018 11:48:42 +0200
Subject: [PATCH rdma-core 2/2] verbs: Fix spelling mistakes in
 ibv_create_flow_action.3

Trivial fix to spelling mistake in ibv_create_flow_action.3 man page
(found by lintian).

Signed-off-by: Benjamin Drung <benjamin.drung@profitbricks.com>
---
 libibverbs/man/ibv_create_flow_action.3.md | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/libibverbs/man/ibv_create_flow_action.3.md b/libibverbs/man/ibv_create_flow_action.3.md
index 20a7b906..7e1f025c 100644
--- a/libibverbs/man/ibv_create_flow_action.3.md
+++ b/libibverbs/man/ibv_create_flow_action.3.md
@@ -148,7 +148,7 @@ packets protected by the SA defined in this action.
 The following members of the esp_attr are used only in full offload mode:
 
 *spi*
-:	The value for the ESP Security Paremeters Index.
+:	The value for the ESP Security Parameters Index.
 	It is only used for *IB_UVERBS_FLOW_ACTION_ESP_FLAGS_FULL_OFFLAOD*.
 
 *seq*
@@ -229,7 +229,7 @@ if (!overlap) {
 }
 ```
 
-This mechanism is controled by the *esp_attr* flag:
+This mechanism is controlled by the *esp_attr* flag:
 
 *IB_UVERBS_FLOW_ACTION_ESP_FLAGS_ESN_NEW_WINDOW*
 : 	This flag is only used to provide stateless ESN support for inline crypto.
@@ -268,7 +268,7 @@ struct ibv_flow_action_esp_aes_keymat_aes_gcm {
 
 *iv_algo*
 :	The algorithm used to transform/generate new IVs with
-	*IB_UVERBS_FLOW_ACTION_ESP_FLAGS_FULL_OFFLOAD* encyption.
+	*IB_UVERBS_FLOW_ACTION_ESP_FLAGS_FULL_OFFLOAD* encryption.
 
 	The only supported value is *IB_UVERBS_FLOW_ACTION_IV_ALGO_SEQ* to
 	generate sequantial IVs.
-- 
2.17.0

